introduction.md

```diff
@@ -0,0 +1,9 @@
+Hello, my name is Hendra Wijaya Djiono.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Reply:
I already added my favorite Code School path... But it can't seem pass the requirement of the quiz